src/Controller/ProxyController.php

```diff
@@ -253,11 +253,7 @@ public function downloads(Request $request): JsonResponse
         }
 
         $this->messageBus->dispatch(new AddDownloads(
-            \array_map(function (array $data): Package {
-                return new Package($data['name'], $data['version']);
-            }, \array_filter($contents['downloads'], function (array $row): bool {
-                return isset($row['name'], $row['version']);
-            })),
+            \array_map(fn(array $data): Package => new Package($data['name'], $data['version']), \array_filter($contents['downloads'], fn(array $row): bool => isset($row['name'], $row['version']))),
```

Comment:
It reads bad, could this be reformatted?
```suggestion
            \array_map(
                fn(array $data): Package => new Package($data['name'], $data['version']),
                \array_filter($contents['downloads'], fn(array $row): bool => isset($row['name'], $row['version']))
            ),
```

Both of the arrow functions could be marked as static (`static fn`), too, but that's just nitpicking.